docs/metadata.rst

```diff
@@ -130,26 +258,27 @@ Examples
 As an example here is a schema using the ``struct`` codec which could apply, for example,
 to the individuals in a tree sequence:
 
-.. code-block:: json
-
-    {
-      "codec": "json",
-      "type": "object",
-      "properties": {
-        "accession_number": {"type": "number", "binaryFormat": "i"},
-        "collection_date": {
-          "name": "Collection date",
-          "description": "Date of sample collection in ISO format",
-          "type": "string",
-          "binaryFormat": "10p",
-          "pattern": "^([1-9][0-9]{3})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])?$"
-        },
-      },
-      "required": ["accession_number"],
-      "additionalProperties": false,
-    }
-
-This schema states that the metadata for the each row of the table
+.. code-block:: python
+
+    schema = metadata.MetadataSchema(
+        {
+            "codec": "struct",
+            "type": "object",
+            "properties": {
+                "accession_number": {"type": "number", "binaryFormat": "i"},
```

Comment:
shouldn't this be `integer`?

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/741/commits/8b80596a531e6a6e0ef8cbe11ae0302e38794a3f